.github/workflows/centraldb_intergration_test.yaml

```diff
@@ -26,6 +30,7 @@ jobs:
 
     - name: Build CentralDashboard Image
       run: |
+        TAG=${{env.TAG}}
```

Comment:
I don't understand what this is doing.

Reply:
It makes sure the image build, and stored in docker, will use the TAG defined in the `env` and not the default one which is
https://github.com/kubeflow/kubeflow/blob/master/components/centraldashboard/Makefile#L2

Reply:
I was trying to say that this is a redundant line, as the global `env` affects all job steps.

Reply:
I meant that this line is redundant because all job steps get all the global `env` exported into them.